src/techs/techs.service.ts

```diff
@@ -353,7 +366,8 @@ export class TechsService {
         techStackCategoryId: number,
         updateTechStackCategoryDto: UpdateTechStackCategoryDto,
     ) {
-        const permission = await this.userCanChangeCategory(
+        const permission = true;
```

Comment:
how come permission is set to true but then a few lines below there is code to throw an error when permission is not truthy?

Reply:
oops - I did this as a shortcut when refactoring and forgot about it...